shinysdr/test/plugins/test_psk31.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2014, 2015, 2016, 2017 Kevin Reid <kpreid@switchb.org>
+#
+# This file is part of ShinySDR.
+# 
+# ShinySDR is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# 
+# ShinySDR is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+# 
+# You should have received a copy of the GNU General Public License
+# along with ShinySDR.  If not, see <http://www.gnu.org/licenses/>.
+
+from __future__ import absolute_import, division
+
+from twisted.trial import unittest
+
+from zope.interface.verify import verifyObject
+
+from shinysdr.plugins.psk31 import PSK31Demodulator
+from shinysdr.interfaces import IDemodulator
+
+
+class TestPSK31Demodulator(unittest.TestCase):
+    def setUp(self):
+        self.demodulator = PSK31Demodulator('PSK31', 48000, None)
+
+    def test_interface(self):
```

Comment:
Seems I can't do that unless I make it enabled by default. Otherwise the plugin isn't registered and the tests fail to find it.